src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
What you are doing now is hiding the actual loudness of the tracks. If a track is mastered with the samples being close to 1 then Mixxx should show this. Your solutions tries to hide how a track was mastered. That enables you to use the full range of the gain knob yes but I don't like that much, considering that I now would be very hard for me to judge at which "loudness" a track is intended to be played. 

Another factor is what @rryan mentioned to make the transition for others easier. So if most other DJ software just pass tracks through without an additional "boost" then we should do the same. People are used to it and it will make it easier for them to try and use Mixxx.

I'm agreeing with @rryan and @ywwg. Your solution is good for a music player like foobar2000 but I don't think it is a good solution for Mixxx.